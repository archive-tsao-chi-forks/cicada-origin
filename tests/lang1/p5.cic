{
  id = (x) => x : (Nat) -> Nat
  (_) => (_) => id(id(id(id(100)))) : (Nat) -> (Nat) -> Nat
}
