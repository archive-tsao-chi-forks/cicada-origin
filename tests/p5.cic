@def xyh "Xie Yuheng"

@show xyh
