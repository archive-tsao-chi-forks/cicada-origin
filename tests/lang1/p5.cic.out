(_1) => (_2) => 666: (Nat) -> (Nat) -> Nat
